docs/rjmc.rst

```diff
@@ -56,9 +59,11 @@ Geometry utility classes
     :nosignatures:
     :toctree: api/generated/
 
-    PredAtomTopologyIndex
-    BootstrapParticleFilter
     GeometrySystemGenerator
-    GeometrySystemGeneratorFast
-    PredHBond
-    ProposalOrderTools
+    .. JRG: These classes does not exist anymore?
+    .. BootstrapParticleFilter
+    .. GeometrySystemGeneratorFast
+    .. PredAtomTopologyIndex
+    .. PredHBond
+    .. .. This one is also dead code in some modules!
+    .. ProposalOrderTools
```

Comment:
i mostly rewrote `geometry.py` and never updated documentation. should there be a .rst for every API-containing `.py` script in the code?

Reply:
That would be one way. The current approach is a semi-automated way to approach this, but it should be rearranged into a different directory tree. I like the `api` stuff to be separate from the user-oriented docs. I can propose a new scheme, but first we need to trim out all the dead leaves :)

If you can add comments to those classes that are gone in the current version and identify if the missing references (check the logs on `CI / Build documentation`) are a bug or just the byproduct of dead code, then I can work on the rest! Thanks!